packages/core/src/external/aws/lambda-logic/document-uploader.ts

```diff
@@ -80,6 +80,7 @@ export async function documentUploaderHandler(
     const docRef = await forwardCallToServer(cxId, apiServerURL, fileData);
     const stringSize = size ? size.toString() : "";
     const hash = eTag ? eTag : "";
+    const mimeType = contentType ? contentType : "";
```

Comment:
Why empty strings instead of undefined? This is just a workaround instead of actually fixing the issue, either:
1. Make it optional on the downstream functions we're calling (createAndUploadMetadataFile and createExtrinsicObjectXml); OR
2. Error and don't accept a file without that information.

I think we should avoid these, it just move issues under the rug.